content/docs/guides/neon-identity.md

```diff
@@ -124,12 +124,41 @@ async function createTodo(userId: string, task: string) {
 }
 ```
 
-## Getting Started
+## Getting started
 
 1. From the Neon Console, navigate to the **Identity** tab. Choose your provider and click **Connect**. Currently, only Stack Auth is available for Early Access users.
```

Comment:
THis isn't exactly like this right now, but might as well just change it to `Auth` tab